components/molecules/TopicViewerContent.tsx

```diff
@@ -85,28 +75,30 @@ export default function TopicViewerContent(props: Props) {
         />
       )}
       <Typography className={classes.title} variant="h6">
-        {topic.name}
+        <span>{topic.name}</span>
+        <Chip
+          label={formatInterval(0, topic.timeRequired * 1000) || "10秒未満"}
+        />
```

Comment:
時間であること自体は日本語使用者であれば時間の単位が存在するので明らかですが、トピックの学習時間であることは何かしらセマンティクスとして残しておくほうが望ましいかも知れないですね。

Reply:
視覚的にもラベリングとして表示されている必要があると考えるのであれば、やはりこのチップ内に文言を追加することになるかと思いました

```suggestion
        <Chip
          label={`学習時間 ${formatInterval(0, topic.timeRequired * 1000) || "10秒未満"}`}
        />
```